src/treq/client.py

```diff
@@ -341,7 +341,8 @@ def _guess_content_type(filename):
 if not _PY3:
     from StringIO import StringIO
     registerAdapter(_from_file, StringIO, IBodyProducer)
-    registerAdapter(_from_file, file, IBodyProducer)
+    # Suppress lint failure on Python 3.
```

Comment:
I think that `# noqa: F821` already explains that a lint message is suppressed.

Maybe it is work adding a comment like:

```
pycodestyle wants us to use `open()` function instead of `file()` constructor, but here we really need to `file` type.
```